src/handlers/flow/FlowStreamHandler.h

```diff
@@ -171,6 +171,7 @@ class FlowMetricsManager final : public visor::AbstractMetricsManager<FlowMetric
 
 class FlowStreamHandler final : public visor::StreamMetricsHandler<FlowMetricsManager>
 {
+    static constexpr const char *_flow_schema{"flow"};
```

Comment:
hm well can we use this in the metric definitions too then?

Reply:
we could not the way I did. I've just changed it to use on bucket as well